config/config.go

```diff
@@ -0,0 +1,73 @@
+package config
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/BurntSushi/toml"
+)
+
+type Config struct {
+	HomeDir string
```

Comment:
We should be explicit with the `toml:"whatever"` tags, especially since I think underscores as word separators is idiomatic there (right?)

Reply:
As far as I can tell, there is not really a standard naming convention. I have seen camel, snake, and kebab case. The default w/ the toml library is camelCase, but if you would prefer underscores, I can add explicit tags.

Reply:
My personal preference is kebab-case, but there are underscores here: https://github.com/toml-lang/toml#example
Also, just generally, I seem to remember underscores being prevalent in INI files, so I think we should go with that.

Reply:
Er nevermind, git config seems to use camelCase, so maybe that's fine too.